src/tools/rados/rados.cc

```diff
@@ -407,6 +407,7 @@ void dump_name(Formatter *formatter, const librados::NObjectIterator& i, [[maybe
 } // namespace detail
 
 unsigned default_op_size = 1 << 22;
+static const unsigned default_read_omap_max_return = 1 << 10;
```

Comment:
s/default_read_omap_max_return/OMAP_BYTES_PER_REQUSET/

Reply:
unless you want to make this a configurable variable, let's name it with uppercase chars.

Reply:
well, have replaced them